src/Microsoft.Diagnostics.Monitoring.RestServer/Controllers/DiagController.cs

```diff
@@ -157,7 +158,7 @@ public Task<ActionResult> GetGcDump(int? pid)
         }
 
         [HttpGet("logs/{pid?}")]
-        [Produces(ContentTypeEventStream, ContentTypeNdJson)]
+        [Produces(ContentTypeEventStream, ContentTypeNdJson, ContentTypeJson)]
```

Comment:
Wouldn't this need to be applied to all of the actions? Or is it that because there is the Produces attribute here that the pipeline exclude 'application/json' as an allowable result (note that the dotnet-monitor tool already adds a Produces filter for 'application/json')?

https://github.com/dotnet/diagnostics/blob/81c1ed54776ac8add8cfe9f311b9d8dda78a9c8e/src/Tools/dotnet-monitor/Startup.cs#L35

Reply:
I believe we need this here because we add a Produces attribute that supersedes the one added at startup. I tested other endpoints, and they properly returned json.